client/CryptoManager.cpp

```diff
@@ -41,6 +41,10 @@ bool CryptoManager::certsLoaded = false;
 ByteVector CryptoManager::keyprint;
 static CriticalSection g_cs;
 
+unsigned char alpn_protos[] = {
+	3, 'a', 'd', 'c',
```

Comment:
static const unsigned char alpn_protos[] ?

Reply:
Sorry, forgot to add it (haven't coded in C++ for a while).